package/doc/sphinx/source/index.rst

```diff
@@ -67,16 +67,33 @@ literature. Please make sure to also reference any
 Thank you!
 
 
-Contents
-========
+MDAnalysis Basics
+=================
 
 .. toctree::
    :maxdepth: 1
-   :numbered:		
-   
+   :numbered:
+
    ./documentation_pages/overview
    ./documentation_pages/topology
+   ./documentation_pages/working_with_atomgroups
    ./documentation_pages/selections
+   ./documentation_pages/working_with_residues
+   ./documentation_pages/working_with_trajectories
+   ./documentation_pages/coordinate_manipulation
+   ./documentation_pages/bonds_angles_torsions
+   ./documentation_pages/advanced_universe
+   ./documentation_pages/advanced_reader
+   ./documentation_pages/advanced_topology
+
+
+API Reference
+=============
```

Comment:
With the sidebar navigation in #1126 I am pretty sure that we need a single toctree. I would suggest to keep the top-toctree pretty light, essentially

1. Overview
2. User guide 
3. API Reference
4. References

where _User guide_ is what you called "MDAnalysis Basics"